CONTRIBUTING.md

```diff
@@ -509,6 +509,38 @@ The following options are available:
 {{< ui-message color="green" text="The message displayed in the notification.">}}
 ```
 
+### Flexbox-formatted content blocks
+CSS Flexbox formatting lets you create columns in article content that adjust and
```

Comment:
###Add two tables side-by-side in a column
Use CSS Flexbox to add two (or more?) tables side-by-side in a column. Content in a Flexbox table adjusts to the viewable width?
Use the `{{< flex >}} shortcode to create the table. Use....to identify each column in the table?